plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/ndt_infra.py

```diff
@@ -169,6 +170,16 @@ def run_ibdiagnet_verification_command():
     status, cmd_output = execute_generic_command(IBDIAGNET_PORT_VERIFICATION_COMMAND)
     return status
 
+def normalize_port_guid_length(port_guid):
+    '''
+    Normalize port guid len - in need add prefix 0
+    :param port_guid:
+    '''
+    port_guid_len = len(port_guid)
+    if port_guid_len < 18: # in case and we have guid that is short - need to add zero
```

Comment:
why are we ignoring guids longer than 18 symbols?
isn't it a problem that UFM needs to report and fail on it? or at least maybe log that guid

Reply:
guid is a number and it could be 0x887a or 0x000887a
the problem is that in ibdiagnet.net_dump and in ibdiagnet.db_csv the format is not the same ... so need to make them of the same length (max between them).